### DISCUSSION TITLE
[@types/node] error TS2307: Cannot find module 'readline/promises' or its corresponding type declarations.

### DISCUSSION THREAD

**Discussion by aminnairi** [DefinitelyTyped/DefinitelyTyped#58019] [Issues with a @types package]
I'm trying to import the `readline/promises` builtin module but the compiler does not recognizes this module altough in Node 17.0.0 and later it works just fine.

Is there something I'm missing?

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
@aminnairi 
it looks there is no `readline/promises.d.ts` module implementation in DT yet (async version). Implementation on DT is up for grab

**Comment by nickserv** [comment]
Looks like this has been fixed.
